### DISCUSSION TITLE
Bard Text Editor - Edit html?

### DISCUSSION THREAD

**Discussion by flymke** [statamic/cms#4391] [Q&A] [answered]
Can I use to html option in bard to also _edit_ html? From what I see I can only _see_ the html code that was generated.
Is there any other option to give clients an easy wysiwyg editor (not markup)?
Bard looks really nice, I am just irritated that I cannot edit the html.

**Comment by duncanmcclean** [comment, marked as answer]
No, the code generated by Bard can't be edited in Bard. If you add sets though, then you can customise the HTML used to display text, images, etc. 

Without sets, Bard will generate HTML itself like you're seeing.

**Reply by flymke** [reply]
Hi there, cool thanks. Do you know if there is any other editor for simple html / wysiwyg type of fields? I have some clients that are used to e.g. TinyMCE where you can simply switch from Text to HTML and vice versa. I know this is probably not intended by bard/statamic or Markup language is better etc. - but I am still a bit irritated that there is no editor like this.

**Reply by duncanmcclean** [reply]
There's a [TinyMCE addon](https://statamic.com/addons/mity-digital/tinymce-cloud) someone's created and put on the Marketplace, maybe it'll help?

**Reply by flymke** [reply]
Oh cool. Thanks ! :-) I will give that a try.
